src/discord-api.ts

```diff
@@ -272,3 +286,14 @@ export const fetchAttachment = async (ctx: Context) => {
 		return new Response(error, { status: 500 })
 	}
 }
+
+export const getLfgTopics = async (ctx: Context) => {
+	const env = ctx.env
+
+	const activeTopics = await getAllTopics(env, LFG_CHANNEL_ID)
+	const archivedTopics = await getAllArchivedTopics(env, LFG_CHANNEL_ID)
+
+	const topics = [...activeTopics, ...archivedTopics]
```

Comment:
Shouldn't you still parse these using the discord message parser to allow the front-end to consume them more easily?

Reply:
It's only the title we need from the topic, I don't think there is need to be parsed.